src/main/webapp/app/exercises/quiz/shared/questions/drag-and-drop-question/drag-and-drop-question.component.html

```diff
@@ -109,7 +111,7 @@ <h4 class="question-title-display">
                             <div
                                 class="drop-location results"
                                 [ngClass]="{
-                                    incorrect: !isLocationCorrect(dropLocation) && !dropLocation.invalid && !invalidDragItemForDropLocation(dropLocation) && !question.invalid
+                                    incorrect: isLocationCorrect(dropLocation) === 1 && !dropLocation.invalid && !invalidDragItemForDropLocation(dropLocation) && !question.invalid
```

Comment:
You are referring to the enum `MAPPED_CORRECT`, right? 
You can also checked directly against the enum, i.e. changing it to:
```
incorrect: isLocationCorrect(dropLocation) === MappingResult.MAPPED_CORRECT
```
if you add, what I suggest in this comment.
This is more explicit and readable.

Reply:
Fixed immediately :)